websites/A/Anizium/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "708746139092123679",
+		"name": "Mert^^#0123"
+	},
+	"service": "Anizium",
+	"description": {
+		"en": "Anizium is a website where you can watch anime with 4K Turkish subtitles and dubs."
+	},
+	"url": "anizium.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/Z3G1Ubm.png",
```

Comment:
Logos should be [512x512px](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).

Reply:
i did ty for everything

Reply:
It doesn't appear that you have resolved these issues yet.